probe/endpoint/connection_tracker.go

```diff
@@ -29,6 +30,9 @@ type connectionTracker struct {
 	flowWalker      flowWalker // Interface
 	ebpfTracker     *EbpfTracker
 	reverseResolver *reverseResolver
+
+	ebpfFailureCount int
```

Comment:
Do we need counts here? We only check it to be higher than 0, right? then we can simply call it `didEBPFFail`?